mix.exs

```diff
@@ -102,10 +107,45 @@ defmodule Boombox.Mixfile do
     ]
   end
 
+  defp aliases do
+    [docs: [&generate_stable_examples/1, "docs", &remove_stable_examples/1]]
+  end
+
+  defp generate_stable_examples(_) do
+    livebook_lines =
+      File.read!("examples.livemd")
+      |> String.split("\n")
+
+    File.rename!("examples.livemd", "_examples.livemd")
```

Comment:
Instead of renaming, I'd copy to `#{Mix.Project.build_path()}/examples.livemd` and operate on that.